backend/src/main/java/sysc4806group25/monkeypoll/controller/ChatController.java

```diff
@@ -24,7 +28,11 @@ public ChatController(VertexAiGeminiChatModel chatModel) {
     }
 
     @PostMapping("user/ai/generate")
+    @HystrixCommand(fallbackMethod = "handleAIProcessingError", commandProperties = {@HystrixProperty(name = "circuitBreaker.requestVolumeThreshold", value = "3"), @HystrixProperty(name = "circuitBreaker.sleepWindowInMilliseconds", value = "50000")})
```

Comment:
![image](https://github.com/user-attachments/assets/f89e8c02-0db1-48ad-aaab-00fe559fd1ef)
Everything looks good! One thing is that we have to change the waiting time for Hysterix, the default timeout period is 500ms but Gemini takes an average of 3 seconds. Could you look into adding a timeout?